CHANGELOG.md

```diff
@@ -42,6 +42,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - [playback] `alsamixer`: make `--volume-ctrl {linear|log}` work as expected
 - [playback] `alsa`, `gstreamer`, `pulseaudio`: always output in native endianness
 - [playback] `alsa`: revert buffer size to ~500 ms
+- [playback] `pulseaudio`: better error handling
```

Comment:
If you're rolling this out for pretty much all of the backends, then you can wrap this up into a single entry.
e.g. when your Alsa PR is merged soon.

Reply:
So something like:
```
- [playback] `alsa`,`pulseaudio`, `pipe`: better error handling

```

I plan on doing what I can. I can't promise *all* the backends,lol!!!

Reply:
As you merge them I will rebase and add them to the lists. Or alternatively as you merge them you can touch up the change log as I don't know what order you plan on merging the 3 open PR's?

Reply:
Sure can do either way.

Reply:
I just removed that line. I will do the same for the pipe PR as soon as I address you comment there.